src/common/java/net/minecraftforge/gradle/common/util/runs/EclipseRunGenerator.java

```diff
@@ -93,7 +97,69 @@ protected Map<String, Document> createRunConfiguration(@Nonnull final Project pr
             }
             javaDocument.appendChild(rootElement);
         }
-        documents.put(runConfig.getTaskName() + ".launch", javaDocument);
+
+        final String configName = (mc.getGenerateRunFolders().getOrElse(false) ? runConfig.getFolderName() + " - " : "") + runConfig.getTaskName() + ".launch";
+        final boolean copyResources = mc.getCopyIDEResources().getOrElse(false);
+
+        if (mc.getEnableEclipsePrepareRuns().getOrElse(false) || copyResources) {
+            final String launchConfigName = project.getName() + " - " + runConfig.getTaskName() + "Slim";
+            documents.put(".eclipse/configurations/" + launchConfigName + ".launch", javaDocument);
```

Comment:
This prefixes with a folder while the others do not. Is this proper?
My assumption would be you do it this way as otherwise the launch files go into the root folder of the project (from memory).
Does this mean they will automatically be imported into Eclipse without any user interaction?

Reply:
Eclipse run configurations are global. The project name prefix makes the launch group know exactly what configurations to launch.
The slim and gradle configurations are in the `.eclipse/configurations` folder because they shouldn't be manually launched and to reduce the files in the root dir. 
Also eclipse scans the entire project for run configurations, so manual interaction is not needed no.

Reply:
Is this folder used by eclipse or something you made up?

Reply:
No it is not, by default eclipse dumps configurations in the root project dir, but it scans the entire project for configurations.